python/pyspark/ml/tests/connect/test_parity_torch_data_loader.py

```diff
@@ -0,0 +1,52 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import shutil
+import tempfile
+import unittest
+from pyspark.sql import SparkSession
+
+have_torch = True
+try:
+    import torch  # noqa: F401
+except ImportError:
+    have_torch = False
+
+from pyspark.ml.torch.tests.test_data_loader import TorchDistributorDataLoaderUnitTests
+
+
+@unittest.skipIf(not have_torch, "torch is required")
```

Comment:
Is this required? Since we have downloaded PyTorch as part of the testing requirements.

Reply:
The other torch distributor tests also have similar annotation, I think it is for some cases without the torch installed.